static/app/chartcuterie/discover.tsx

```diff
@@ -76,11 +76,23 @@ discoverCharts.push({
   getOption: (
     data: {stats: Record<string, EventsStats>} | {seriesName?: string; stats: EventsStats}
   ) => {
-    if (isArray(data.stats.data) && data.stats.data.length === 0) {
+    if (isArray(data.stats.data)) {
+      const color = theme.charts.getColorPalette(data.stats.data.length - 2);
```

Comment:
Can `data.stats.data.length` ever be 0? If so is it going to cause problems with ECharts if `color` is `undefined`?

Reply:
> Can `data.stats.data.length` ever be 0? If so is it going to cause problems with ECharts if `color` is `undefined`?

@Zylphrex I think it should be fine? [Deubgging](https://github.com/getsentry/sentry/blob/master/src/sentry/web/frontend/debug/debug_chart_renderer.py#L130) it wouth data.length==0 returns these:
![Screen Shot 2021-08-24 at 10 27 44 AM](https://user-images.githubusercontent.com/63818634/130635178-6a1b1467-e055-4cbf-81d1-67fb170877fb.png)

Reply:
And if it had data like this:
```discover_empty = {
    "seriesName": "Discover empty",
    "stats": {
        "data": [
            [1629638700, [{"count": 0}]],
            [1629639000, [{"count": 0}]],
        ],
    },
}
```

it will now look like:
![Screen Shot 2021-08-24 at 10 30 51 AM](https://user-images.githubusercontent.com/63818634/130635573-1aa72105-00bc-429a-b7be-5854d5bad2ff.png)